src/Console/Command/ConfigureCommand.php

```diff
@@ -24,31 +24,18 @@ class ConfigureCommand extends Command
 {
     const COMMAND_NAME = 'configure';
 
-    /**
-     * @var GrumPHP
-     */
+    /** @var GrumPHP */
     protected $config;
 
-    /**
-     * @var Filesystem
-     */
+    /** @var Filesystem */
     protected $filesystem;
 
-    /**
-     * @var Repository
-     */
+    /** @var Repository */
     protected $repository;
 
-    /**
-     * @var InputInterface
-     */
+    /** @var InputInterface */
     protected $input;
 
-    /**
-     * @param GrumPHP    $config
```

Comment:
I would also leave it as it is, since we still use PHP5.
_This actually counts for every removed docblock in your PR._